packages/kbn-plugin-generator/sao_template/template/i18nrc.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Filename must be named `.i18nrc.json`. the template is naming them `i18nrc` without the `.` at the beginning.

Reply:
It's renaming it during the `post` phase (it's a `sao` limitation)